Mod2/Preparation/Week2/TDD.md

```diff
@@ -1 +1,12 @@
-## Pre-Work for [TDD]()
\ No newline at end of file
+## Pre-Work for [TDD](/Mod2/Lessons/Week2/TDD.md)
```

Comment:
Ooooo I really like this preparation of thinking about the real world impact of bugs. Great find on the article!